tests/TDD_tests.py

```
@@ -0,0 +1,126 @@
+                    },
+                },
+                {
+                    "project": "project-XXXXXXXXX",
+                    "id": "file-BBBBBBBBBBB",
+                    "describe": {
+                        "id": "file-BBBBBBBBBBB",
+                        "project": "project-XXXXXXXXX",
+                        "class": "file",
+                        "name": "run.A_RUN_NAME.lane.all_001.tar.gz",
+                        "folder": "/fake_runfolder_01/runs",
+                        f"created": {now},
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)